examples/server-communication__visit-2nd-domain/cypress/integration/spec.js

```diff
@@ -0,0 +1,32 @@
+/// <reference types="cypress" />
+describe('Two domains', () => {
+  it('visits 1nd domain', () => {
+    cy.visit('https://www.cypress.io/')
+    // there are several GitHub links on the page, make sure
+    // to use the selector that returns a single item
+    cy.get('header [aria-label="Check out our github page"]')
+    .should('have.length', 1)
+    // from the jQuery wrapping <a href="https://github.io ..." />
+    // get the "href" value
+    .invoke('attr', 'href')
+    .then((url) => {
+      // save the value in the plugins process
+      // that is preserved between test and browser restarts
```

Comment:
It's not really a test or browser restart, but the fact that we're resetting the host URL because it's cross-domain which cleans the state. You wouldn't need to use cy.task() at all if it were the same domain URL.

Reply:
well, we reload the browser window if we detect domain change, can update the comment

Reply:
yeah - 'browser reload' would be the right word, not 'browser restarts' 👍